cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,3 +1,7 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+{% set e2e_opts = '' -%}
+{% if pillar.get('e2e_test_environment', '').lower() == 'true' -%}
+{% set e2e_opts = '-s devicemapper' -%}
```

Comment:
Extended attributes (=`devicemapper` storage driver)  are necessary to run Gluster and Ceph containers. I guess I could create a 1GB file *inside* the containers, format it as ext3 and create Gluster/Ceph bricks there, but it looks like an overkill to me.

IMO it's not up to Kubernetes to decide what storage backend the system admin configures for Docker, it should work with all of them. Just for e2e tests it must be something that supports extended attributes inside the containers.